examples/asgilook/asgilook/store.py

```diff
@@ -3,9 +3,10 @@
 import io
 
 import aiofiles
-import falcon
 import PIL.Image
 
+import falcon
```

Comment:
On the one hand, this makes sense from Falcon's perspective, but from the tutorial's perspective, `falcon` should be grouped just together with other 3rd parties just like `aiofiles`. But maybe it is hard to reconfigure Ruff just for the tutorials? Regardless, we need to make sure that other copypasted incarnations of this file are synced.